Src/WitsmlExplorer.Frontend/components/Sidebar/ServerManager.tsx

```diff
@@ -1,6 +1,5 @@
 import { useIsAuthenticated } from "@azure/msal-react";
```

Comment:
The ServerManager file should be moved to the ContentViews directory now that it is no longer placed in the Sidebar.